### ISSUE TITLE
Elements page fails Lighthouse accessibility checks

### ISSUE THREAD

**Issue by chosak** [cfpb/design-system#2361]
**Describe the bug**
The [Elements page](https://cfpb.github.io/design-system/components/elements) fails Lighthouse accessibility checks. See report [here](https://github.com/cfpb/design-system/actions/runs/17449158887/job/49550317270).

**Screenshots**
<img width="814" height="624" alt="Image" src="https://github.com/user-attachments/assets/817dfd4c-f8c9-4048-885b-534c2811c5d9" />

**Additional context**
Looks to be added in #2337 by @anselmbradford.

I note that this page isn't linked from the Design System sidebar, is that intentional?

**Comment by chosak** [comment]
@anselmbradford well don't I have egg on my face! What do you think about the original accessibility failure?

**Comment by anselmbradford** [comment]
> @anselmbradford well don't I have egg on my face! What do you think about the original accessibility failure?

I have it fixed, but it's mixed up in a branch where I have other work going on. I'll spin it out to its own PR if too much time passes.
